src/selection-table/SelectionTable.css

```diff
@@ -30,6 +30,59 @@
     padding: 0;
 }
 
-.query-entry h3{
+.query-entry *{
+    margin: 0;
+        margin-top: 5px;
+
+}
+
+.extend-container{
+    display: flex;
+    margin: 0;
+    margin-top: 10px;
+    padding: 0;
+}
+
+.extend-container *{
+    display: inline-block;
+}
+
+.query-entry .query-selector{
     user-select: none;
+}
+
+.query-entry .query-selector:hover{
+    user-select: none;
+    cursor: pointer;
+}
+
+.arrow {
+    padding: 0;
+    float: right;
+    margin-left: auto;
+}
+  
+  .arrow::before {
+    position: absolute;
+    content: '';
+    width: 0;
+    height: 0;
+    border: .5em solid transparent;
+    border-left-color: gray;
+    transform-origin: 0 50%;
+    transition: transform .25s;
+  }
+  
+  .arrow.down::before {
+    transform: rotate(90deg);
+    transition: transform .25s;
+  }
```

Comment:
Fix formatting.

Reply:
This, among other things is resolved in [commit a812905](https://github.com/SolidLabResearch/generic-data-viewer/pull/3/commits/a8129055504bbe5471cbfdfe3a7b882e857b70fd)